.circleci/config.yml

```diff
@@ -18,7 +15,7 @@ parameters:
     default: '14.0'
   upgrade_pg_versions:
     type: string
-    default: '12.8-13.4-14.0'
+    default: '13.4-14.0'
```

Comment:
We require a docker image with a tag that is identical to this field. Did you create a new docker image for this to be valid?

Reply:
I created one in https://github.com/citusdata/the-process/pull/72